app/js/main.js

```diff
@@ -254,6 +258,26 @@ function startFlowAuto() {
         location.reload();
       });
 
+      // If the publisher is unlocking the page for their own reason,
+      // please let google know here.
+      if (OWNS_PUBLISHER_SUBSCRIPTION) {
+        subscriptions.setShowcaseEntitlement({
+          isUserRegister: true,
+          entitlement: 'EVENT_SHOWCASE_UNLOCKED_BY_SUBSCRIPTION',
+        });
+      } else if (USED_PUBLISHER_METER) {
+        subscriptions.setShowcaseEntitlement({
+          isUserRegister: true,
+          entitlement: 'EVENT_SHOWCASE_UNLOCKED_BY_METER',
+        });
+      } else if (IS_FREE_ARTICLE) {
+        subscriptions.setShowcaseEntitlement({
```

Comment:
`swg-js` automatically unlocks free articles. Would this API introduce a new responsibility for publishers to call this on every free article?

Reply:
If you can point out where swg-js knows it's a free article, I can mark it there so it's not a publisher responsibility

Reply:
I'm not sure where exactly it happens

Reply:
Per John swg-js doesn't know this and the publisher does.  I've got some other improvements coming though